README.md

```diff
@@ -20,6 +20,9 @@ born to bring that same vision to the other platforms supported by Ebitengine.
 - **Smaller Binaries**: Using Cgo generates a C wrapper function for each C function called. Purego doesn't!
 - **Dynamic Linking**: Load symbols at runtime and use it as a plugin system.
 - **Foreign Function Interface**: Call into other languages that are compiled into shared objects.
+- **Cgo Fallback**: Works even with CGO_ENABLED=1 so incremental porting is possible. 
+This also means unsupported Linux architectures (riscv64, mips, etc.) will still work
```

Comment:
What about unsupported FreeBSD architectures? If this is the same, what about 

```
This also means unsupported architectures (linux/riscv64, linux/mips, etc.) will still work...
```

?

Reply:
Interestingly, FreeBSD/RISCv64 is supported by Go

```
$ go tool dist list
aix/ppc64
android/386
android/amd64
android/arm
android/arm64
darwin/amd64
darwin/arm64
dragonfly/amd64
freebsd/386
freebsd/amd64
freebsd/arm
freebsd/arm64
freebsd/riscv64
illumos/amd64
ios/amd64
ios/arm64
js/wasm
linux/386
linux/amd64
linux/arm
linux/arm64
linux/loong64
linux/mips
linux/mips64
linux/mips64le
linux/mipsle
linux/ppc64
linux/ppc64le
linux/riscv64
linux/s390x
netbsd/386
netbsd/amd64
netbsd/arm
netbsd/arm64
openbsd/386
openbsd/amd64
openbsd/arm
openbsd/arm64
plan9/386
plan9/amd64
plan9/arm
solaris/amd64
wasip1/wasm
windows/386
windows/amd64
windows/arm
windows/arm64
```

Reply:
I haven't tested it but I don't see why it wouldn't work.